tests/api_e2e/analytics.test.js

```diff
@@ -28,16 +30,21 @@ describe('check basic analytics query', () => {
         expect(response.statusCode).toBe(401)
     })
 
+    it('should return status 401 when sending query without podcast id', async () => {
```

Comment:
maybe we can add a check if podcast id != account id

Reply:
good idea; added